GliaWidgets/SecureConversations/SecureConversations.WelcomeViewModel.swift

```diff
@@ -31,25 +31,43 @@ extension SecureConversations {
 }
 
 extension SecureConversations.WelcomeViewModel {
-    func props() -> SecureConversations.WelcomeViewController.Props {
+    typealias Props = SecureConversations.WelcomeViewController.Props
+    typealias WelcomeViewProps = SecureConversations.WelcomeView.Props
+    func props() -> Props {
         let welcomeStyle = environment.welcomeStyle
-        let filePickerButtonTap = Cmd { print("### file picker") }
-        let props: SecureConversations.WelcomeViewController.Props = .welcome(
+        let filePickerButton = SecureConversations.WelcomeView.Props.FilePickerButton(
+            isEnabled: true,
+            tap: Cmd { print("### file picker") }
+        )
+
+        let messageLenghtWarning = messageText.count > Self.messageTextLimit ? welcomeStyle
+            .messageWarningStyle.messageLengthLimitText
+            .withTextLength(String(SecureConversations.WelcomeViewModel.messageTextLimit))
+        : ""
+
+        let warningMessage = WelcomeViewProps.WarningMessage(
+            text: messageLenghtWarning,
+            animated: true
+        )
+
+        let sendMessageButton: WelcomeViewProps.SendMessageButton = .loading//.disabled//.active(Cmd { print("### send message") })
```

Comment:
Is this for testing?

Reply:
Yes. Updated c88376711489034aa158c21d3c4e5b3fc04080e4.